### PR TITLE
Notebook update, testing

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

File: .github/workflows/test-notebooks.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  nbmake:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 3
+      matrix:
+        os: [ubuntu-22.04, macos-12, windows-2022]
+
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-python@v3
```
Comment (jhlegarreta) [inline .github/workflows/test-notebooks.yml:15]: Thanks for the effort Matt. We've bumped `actions/setup-python@v4` across several repositories in the last few days.

Eventually, this workflow could also be transferred to `ITKRemoteModuleBuildTestPackageAction` @tbirdso ?

Comment (tbirdso) [reply .github/workflows/test-notebooks.yml:15]: Fully in favor of this 👍

Comment (thewtex) [reply .github/workflows/test-notebooks.yml:15]: Thanks for the review @jhlegarreta !

Done.

I like the idea of consolidating the configuration.

**Review by jhlegarreta — APPROVED** [review]

File: .github/workflows/test-notebooks.yml (hunk 1 above)
Comment (jhlegarreta) [reply .github/workflows/test-notebooks.yml]: Resolving.

**Pull request merged**